katas/content/complex_arithmetic/complex_powers_real/Solution.qs

```diff
@@ -1,14 +1,13 @@
-namespace Kata { 
+namespace Kata {
     open Microsoft.Quantum.Math;
 
     function ComplexExpReal(r : Double, x : Complex) : Complex {
         if AbsD(r) < 1e-9 {
-            return Complex(0.0, 0.0);
```

Comment:
Is this no longer recommended? I liked it.

Reply:
I agree that the new way is more verbose, but that's how structs ended up, since the fields are named. I think we will want to discuss adding shorthand syntax.

The old UDT syntax does still work, so feel free to continue using it.

Reply:
We have a newer way to do it, so I was updating to that new syntax. However, yeah, it might be good to keep the katas with the older, more concise syntax until a later date. What do you think?